scripts/variantstore/beta_docs/gvs-outputs.md

```diff
@@ -0,0 +1,126 @@
+| high_CALIBRATION_SENSITIVITY_INDEL             | Sample Genotype FT filter value indicating that the genotyped allele failed INDEL model calibration sensitivity cutoff (0.99) | The VETS filtering score. This is a ##COMMENT in the VCF because this is not applied as a site-level filter. GVS is allele-specific. The FT for an allele will have this FT tag if it hits the CALIBRATION_SENSITIVITY threshold. A site can pass with failing genotypes, however we recommend filtering these genotypes. See example variants below. |
+| EXCESS_ALLELES                                 | Site has an excess of alternate alleles based on the input threshold                                                          | When a site has more than 100 alternate alleles, this filter will be present.                                                                                                                                                                                                                                                                         |
+| OUTSIDE_OF_TARGETS                             | Outside of sequencing target intervals                                                                                        | Exome only. The site is not within the target intervals of the exome assay. We recommend filtering out these sites as we cannot stand behind the quality of sites called outside of the targets.                                                                                                                                                      |
+
+
+### Example sites and how to interpret them
+#### Passing site
+```
+chr20   66369   .       C       T       .       .       AC=2;AF=1.00;AN=2;AS_QUALapprox=0|982;AS_YNG=G;CALIBRATION_SENSITIVITY=0.2492;QUALapprox=982;SCORE=-0.3836      GT:AD:GQ:RGQ    ./.         ./.     1/1:0,29:87:982
+```
+
+#### A site with failing calibration sensitivity SNPs for all sample genotypes
+```
+chr20   62898   .       T       A       .       .       AS_QUALapprox=0|933;AS_YNG=G;CALIBRATION_SENSITIVITY=0.9999;QUALapprox=324;SCORE=-0.7456        GT:AD:FT:GQ:PGT:PID:RGQ 0/1:26,11:high_CALIBRATION_SENSITIVITY_SNP:99:0|1:62818_T_G:384     0/1:23,7:high_CALIBRATION_SENSITIVITY_SNP:99:0|1:62818_T_G:225  0/1:18,9:high_CALIBRATION_SENSITIVITY_SNP:99:0|1:62818_T_G:324
+```
+
+#### Multi-allelic site with one passing and one failed sample genotype
+```
+chr20   245443  .       GTATATATA       GTATATA,G       .       .       AC=0,1;AF=0.00,0.500;AN=2;AS_QUALapprox=0|339|238;AS_YNG=G,G;CALIBRATION_SENSITIVITY=0.9957,0.9402;QUALapprox=238;SCORE=-0.6448,-0.5305     GT:AD:FT:GQ:RGQ 0/2:11,0,9:PASS:99:238  0/1:10,5,0:high_CALIBRATION_SENSITIVITY_INDEL:99:264    0/1:5,3,0:high_CALIBRATION_SENSITIVITY_INDEL:75:75
+```
+
+---
+
+**Warning:**
+The AC and AN numbers we calculate in the VCFs are based on following the filtering recommendations above. This follows GATK best practices.
```

Comment:
Just the FT status or the HQ genotypes filters?  GATK best practices don't really apply here, but you could say it's in accordance with the gnomAD conventions.